specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2023-07-12-preview/AlertsManagement.json

```diff
@@ -0,0 +1,1772 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2023-07-12-preview",
+    "title": "Azure Alerts Management Service Resource Provider",
+    "description": "Azure Alerts Management Service provides a single pane of glass of alerts across Azure Monitor."
```

Comment:
> pane of glass  [](http://example.com/codeflow?start=69&length=15)

'pane of glass' sounds like a trending metaphor for a 'view' or 'overview'? But it might be better to keep the docs less metaphorical.

Reply:
This PR does not modify this.. It just adds the alert enrichments. I already added WI to fix current existing issues in this repo:
Product Backlog Item 24832853: Fix existing swagger Lint errors
Can I work on this when I'll work on this WI?